src/redisearch_rs/ffi/src/lib.rs

```diff
@@ -31,3 +31,9 @@ include!(concat!(env!("OUT_DIR"), "/bindings.rs"));
 
 /// Rust implementation of `t_fieldMask` from `redisearch.h`
 pub type FieldMask = t_fieldMask;
+
+#[cfg(target_arch = "x86_64")]
+pub const RS_FIELDMASK_ALL: FieldMask = u128::MAX;
+
+#[cfg(not(target_arch = "x86_64"))]
+pub const RS_FIELDMASK_ALL: FieldMask = u64::MAX;
```

Comment:
Isn't `t_fieldMask` always a `u128`?

Reply:
Nope, it based off the arch

https://github.com/RediSearch/RediSearch/blob/0c8a28b0ad094119ce32872542b49da9e293a057/src/redisearch.h#L40-L48

Reply:
Not, sure. `bindgen` would have generated the Rust type from the C code. And then `cbindgen` generated that from the Rust type again. I would say it appearing in the `types_rs.h` again is an error as it should only appear in the `redisearch.h` file.